src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -296,13 +295,13 @@ const styles = StyleSheet.create({
     justifyContent: 'center'
   },
   newPostText: {
-    color: COLORS.holytosca,
+    color: COLORS.blue,
     marginRight: dimen.normalizeDimen(11),
     ...FONTS.h3
   },
   animatedViewContainer: (hideBackIcon) => ({
     flexDirection: 'row',
-    backgroundColor: 'white',
+    backgroundColor: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These changes replace hardcoded color values and improve readability by using named constants instead. This is a good practice as it makes the code easier to understand and maintain.
```diff
-    color: COLORS.holytosca,
+    color: COLORS.blue,

-    backgroundColor: 'white',
+    backgroundColor: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->